language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/visitors/FunctionCallExpressionTypeFinder.java

```diff
@@ -424,6 +427,12 @@ public void visit(WhileStatementNode whileStatementNode) {
 
     @Override
     public void visit(ConditionalExpressionNode conditionalExpressionNode) {
+        if (this.functionCallExpressionNode != null &&
+                PositionUtil.isWithinLineRange(this.functionCallExpressionNode.lineRange(),
+                conditionalExpressionNode.lhsExpression().lineRange())) {
```

Comment:
Let's add the `!conditionalExpressionNode.lhsExpression().isMissing()` condition as well